tests/e2e/csi_static_provisioning_basic.go

```diff
@@ -140,6 +142,9 @@ var _ = ginkgo.Describe("Basic Static Provisioning", func() {
 		} else {
 			fullSyncWaitTime = defaultFullSyncWaitTime
 		}
+		vcAddress := e2eVSphere.Config.Global.VCenterHostname + ":" + sshdPort
+		isStorageQuotaFSSEnabled = isFssEnabled(ctx, vcAddress, "STORAGE_QUOTA_M2")
```

Comment:
Please add a comment to remove this once the FSS is enabled

Reply:
done